cfgov/jinja2/v1/_includes/macros/category-icon.html

```diff
@@ -16,7 +16,7 @@
 
    ========================================================================== #}
 
-{% macro render(category, additional_classes) %}
+{% macro render(category, additional_classes, round) %}
```

Comment:
Yes, that makes sense! Updated in https://github.com/cfpb/cfgov-refresh/pull/2955/commits/46d56756d3a15495455bddde7eee8ba98b0eeae1